codecov.yml

```diff
@@ -0,0 +1,15 @@
+comment:
```

Comment:
We can remove this file and put it into the org namespace, see https://docs.codecov.com/docs/codecov-yaml.